Assets/PlayroomKit/PlayroomKit.cs

```diff
@@ -1095,6 +1105,9 @@ private static void InvokeRpcRegisterCallBack(string dataJson, string senderJson
 
         public static void RpcCall(string name, object data, RpcMode mode, Action callbackOnResponse)
         {
+            // setting to sync the event name with other players
```

Comment:
This is called a "parrot comment"; the comment is describing the same thing the code describes. Usually, a comment should be more of the "Why?" than the "What?". The What can be understood just by reading the code.